packages/typespec-test/test/contentsafety_modular/generated/typespec-ts/review/ai-content-safety.api.md

```diff
@@ -18,13 +23,18 @@ export interface AddOrUpdateBlockItemsResult {
     value?: TextBlockItem[];
 }
 
+// @public
+export interface AnalyzeImageOptions {
+    categories?: ImageCategory[];
+    image: ImageData_2;
```

Comment:
Could you help me understand it is `ImageData_2`?

Reply:
see https://github.com/Azure/azure-sdk-for-js/pull/26542#discussion_r1327118415 and https://github.com/Azure/azure-sdk-for-js/pull/26542#discussion_r1327869367